sxbp/saxbospiral.h

```diff
@@ -41,8 +41,6 @@ typedef struct sxbp_version_t {
 } sxbp_version_t;
 
 extern const sxbp_version_t LIB_SXBP_VERSION;
-// flag for whether PNG output support has been compiled in based, on macro
-extern const bool SXBP_PNG_SUPPORT;
```

Comment:
This was removed from this compilation unit and moved to `render_backends/backend_png` as this was deemed a more appropriate place for it. As far as I know, this may break API compatibility but not ABI compatibility.